litellm/tests/test_bedrock_completion.py

```diff
@@ -205,6 +206,232 @@ def test_completion_bedrock_claude_sts_client_auth():
     except Exception as e:
         pytest.fail(f"Error occurred: {e}")
 
+@pytest.fixture()
+def bedrock_session_token_creds():
+    print("\ncalling oidc auto to get aws_session_token credentials")
+    import os
+
+    aws_region_name = os.environ["AWS_REGION_NAME"]
+    aws_session_token = os.environ.get("AWS_SESSION_TOKEN")
```

Comment:
is this line valid? 

`os.environ.get()` ? 

if you can share a screenshot of your test passing this pr should be good to merge @bschulth

Reply:
os.environ supports the same semantics as a python dict.  You can use the .get() function to optionally retrieve a value with out an exception if it is not set, or optionally provide a default value:
1. if the key does not exist, it returns None rather than throwing an exception
2. you can also provide a default value
3. same...default value test
4. if you use square brackets, it throws an exception if the key does not exist:
<img width="538" alt="image" src="https://github.com/BerriAI/litellm/assets/770344/d61054e8-db8e-401f-8a3a-e36cd0095d73">

So in the code above, we expect that items using square bracket notation always exist, or throw an exception.  But because a user may not always have an aws_session_token, we use get() accessor so that it returns None if not present.

Here is a screenshot of the 2 tests passing:
<img width="1383" alt="image" src="https://github.com/BerriAI/litellm/assets/770344/35ebfe7f-d01a-4a71-a285-1a211014a5fd">